config/routes.rb

```diff
@@ -309,4 +309,6 @@
 
   match ':controller(/:action(/:id))(.:format)'
 
+  match 'embed/:tagname' => 'notes#embed
```

Comment:
and if you change it to `gridEmbed`, this'll have to change too, although perhaps we could make the URL something like `grid/:tagname` for simplicity?